plugins/polio/export_utils.py

```diff
@@ -1,6 +1,6 @@
-from openpyxl import Workbook
-from openpyxl.styles import Alignment, Font, PatternFill
-from openpyxl.styles.borders import Border, Side
+from openpyxl import Workbook  # type: ignore
+from openpyxl.styles import Alignment, Font, PatternFill  # type: ignore
+from openpyxl.styles.borders import Border, Side  # type: ignore
 import datetime as dt
```

Comment:
Maybe we can move that to the mypy config file as discussed earlier?

Reply:
Fixed in:
https://github.com/BLSQ/iaso/pull/73/files#diff-7cc071ededc430f84e05289b1728b43f75f33a5e2e59978bd834cd7b60f9fb85R226